core/domain/learner_progress_services.py

```diff
@@ -464,14 +628,16 @@ def remove_collection_from_completed_list(user_id, collection_id):
 
 
 def _remove_activity_ids_from_completed_list(
-        user_id, exploration_ids, collection_ids):
-    """Removes the explorations and collections from the completed list of the
-    learner.
+        user_id, exploration_ids, collection_ids, story_ids, learnt_topic_ids):
+    """Removes the explorations, collections, stories and learnt_topics
```

Comment:
Remove underscore. Docstring is a description, so you can just mention 'learnt topics'.

Reply:
Done.